data_pipeline/vocab.py

```diff
@@ -2,6 +2,7 @@
 from collections import Counter
```

Comment:
your formatting differs from that of the project -- which makes it seem like there are more changes than there actually are
Please keep the same formatting as in the project and make sure only the relevant changes appear in your PR before opening it